pkg/leeway/reporter.go

```diff
@@ -354,22 +371,56 @@ func (r *HTMLReporter) Report() {
 	vars["RootPackage"] = r.rootPackage
 
 	tmplString := `
-<h1> Leeway build for <code>{{ .RootPackage.FullName }}</code></h1>
-{{ range $pkg, $report := .Packages }}
-<details{{ if $report.HasError }} open{{ end }}><summary> {{ $report.StatusIcon }} <b>{{ $pkg }}</b> - {{ $report.DurationInSeconds }}</summary>
-
-{{ if $report.HasError }}
-<pre>
-{{ $report.Error }}
-</pre>
-{{ end }}
-
-<pre>
-{{ $report.Logs }}
-</pre>
-
+<h1>{{ .RootPackage.FullName }}</h1>
+<p>Leeway built the following packages</p>
+<table>
+	<thead>
+		<tr>
+			<td>🚦 Status</td>
+			<td>📦 Package</td>
+			<td>⏰ Duration</td>
+			<td>🔬 Details</td>
+		</tr>
+	</thread>
+	<tbody>
+		{{- range $pkg, $report := .Packages }}
+		<tr>
+			<td>{{ $report.StatusIcon }}</td>
+			<td>{{ $pkg }}</td>
+			<td>{{ $report.DurationInSeconds -}}</td>
+			<td><a href="#{{ $pkg }}">See below</td>
+		</tr>
+		{{- end }}
+	</tbody>
+<table>
+<p>For details around each package, see below<p>
+{{- range $pkg, $report := .Packages }}
+<h2 id="{{ $pkg }}">{{ $pkg }}</h2>
+{{ if $report.HasError -}}
+<details>
```

Comment:
```suggestion
<details open> 
```

Nit: un-collapse the "details" block by default so that users don't need to do that manually if there is an error.